### PR TITLE
Add tests with Python 3.9 and 3.10

### PR DESCRIPTION
**Pull request by pmderodat**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,9 @@
 e3-testsuite
 ============
 
-Generic testsuite framework in Python. Supports Python 3.8+. Full documentation
-available on [Read the Docs](https://e3-testsuite.readthedocs.io/en/latest/).
+Generic testsuite framework in Python. Supports Python 3.9 and Python 3.10.
```
Comment (liaiss) [inline README.md:4]: Maybe keeping Python 3.9+ is better, to ease the update.

Comment (pmderodat) [reply README.md:4]: I don’t follow: you suggested to stop advertising 3.8 support to make it clear that we don’t support it anymore. Now, when Python 3.11 will be released, we won’t be able to claim that we support it unless we do test it. So if we are to be very precise in the README, I would prefer not advertising support for releases that don’t exist yet: it’s better to update the README at the same time we add the corresponding tests, no?

Comment (pmderodat) [reply README.md:4]: Ping?

Comment (liaiss) [reply README.md:4]: I'm ok with that I will +2

**Review by liaiss — APPROVED** [review]

**Pull request merged**